packages/sandbox-node-vm/src/index.ts

```diff
@@ -25,6 +25,8 @@ export function HybridReadableStream(...args: any[]) {
     },
   })
 }
+// @ts-ignore ho boy I don't want to do this
+global.ReadableStream = WebReadableStream
```

Comment:
What's the issue?

Reply:
Oh just because this is in a "separate" package (it potentially shouldn't be separate) that _happens_ to be included before `readable-stream-node-to-web`, which requires the `global.ReadableStream` to be there.

I think I need to revisit this whole setup...